packages/flutter_tools/templates/app/winuwp.tmpl/project_version

```diff
@@ -0,0 +1 @@
+0
```

Comment:
This is the folder where the rest of the template files need to go. There is also a manifest in the templates/ where they need to be listed.